pkg/loop/internal/broker.go

```diff
@@ -39,7 +39,7 @@ func (a *atomicBroker) DialWithOptions(id uint32, opts ...grpc.DialOption) (conn
 	return a.load().DialWithOptions(id, opts...)
 }
 
-func (a *atomicBroker) NextId() uint32 {
+func (a *atomicBroker) NextId() uint32 { //nolint:revive
```

Comment:
what does this do?

Reply:
Ignores the naming failure on `Id` instead of `ID`, since we are implementing an interface from another package.